fahrschein-metrics-dropwizard/build.gradle

```diff
@@ -5,7 +5,7 @@ plugins {
 
 dependencies {
     api project(':fahrschein')
-    api 'io.dropwizard.metrics:metrics-core:4.2.23'
+    api 'io.dropwizard.metrics:metrics-core:4.2.25'
```

Comment:
https://github.com/dropwizard/metrics/releases  4.2.25 is not tagged in github 👎

Reply:
using 4.2.24